release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 29 May 2024 - Enhancement - Popup buttons [ID_39010]
+
+Buttons in the popups have been updated to the new style.
+
+#### 29 May 2024 - Fix - Admin - email validation [ID_39791]
+
+The email validation has been updated to support all domain extensions.
+
+#### 29 May 2024 - Enhancement - Admin - DMS overview failover pair offline status [ID_39694]
+
+The DMS overview page will now correctly show the failover pair agent as offline when this is the case.
```

Comment:
"Failover" should always be capitalized if it's a reference to the DataMiner Failover feature. Similarly, "Agent" should always be capitalized if it's a reference to a DataMiner Agent.

```suggestion
#### 29 May 2024 - Enhancement - Admin - DMS overview Failover pair offline status [ID_39694]

The DMS overview page will now correctly show a Failover Agent as offline when appropriate.
```